modules/salt/pages/salt-example-formula.adoc

```diff
@@ -49,6 +49,8 @@ salt '$your_client' pillar.items
        foobar
    mgr_server:
        manager_server
+   hub_server:
```

Comment:
This will change.

Reply:
in that case, we'll need to change it in master too.